### ISSUE TITLE
MacOS binary requested

### ISSUE THREAD

**Issue by cwscws** [kavika13/jumpmanzero#34]
As it’s crossplatfoem please release a MacOS binary, too!

Thank you

**Comment by kavika13** [comment]
Will do. Need to do a bit of work on it probably, so it'll take me a few weeks, but I have both an x86 and m1 mac so shouldn't be too difficult.

**Comment by cwscws** [comment]
Wow! Thanx!

I also like to play Jumpman Under Construction as is has an editor. Unfortunately it was never finished and has been abandoned a long time ago…

**Comment by cwscws** [comment]
Any news?

**Comment by kavika13** [comment]
> Any news?

Nah, not yet, sorry. My day job is getting close to a milestone, and it's taking up most of my time right this minute.

I definitely will be coming back to this project, but it'll either be in a few months, or I'll get a wild hair some weekend and just get it working.

**Comment by cwscws** [comment]
I tried it with Parallels Desktop/Windows 1 ARM. 
Absolutely playable!

Only one thing I noticed: when defining keys the cursor keys are not recognized when pressed. But while playing cursor keys are working.

I wish you a happy wild hair day to do the Mac binary soon! 😎
